vp_extraction.py

```diff
@@ -96,6 +101,11 @@ def parse_args():
                         action="store_true",
                         help='''Indicate that the source of the data is
                         the Met office radars''')
+    #######Neely
```

Comment:
It might be a good idea going forward to be introducing a parameter file in the next update, especially with the size of the list of command line arguments. This would mean changing the command-line arguments to be just the name of the file containing a dictionary of the parameters for the run.